torch/csrc/jit/mobile/upgrader_mobile.cpp

```diff
@@ -46,6 +46,14 @@ getOperatorVersionMapForMobile() {
                     std::vector<Upgrader>({
                         Upgrader({0, 3, "div__Tensor_0_3", 3})
                     })},
+                {std::string("aten::linspace"),
+                    std::vector<Upgrader>({
+                        Upgrader({0, 7, "linspace_0_7", 7})
```

Comment:
I'm still confused why linspace has a 0, 7, 7 and linspace.out has 0, 7, 8

Reply:
@cccclai should know better

Reply:
Oh the last 7 here means it's corresponding index in the bytecode list, generated from `getUpgraderBytecodeList()`. It's related to how lite interpreter works. For example, when executing the instruction
```
RUNNING 0 STOREN 1 3
RUNNING 1 DROPR 1
RUNNING 2 LOAD 2
RUNNING 3 LOAD 3
RUNNING 4 CALL 0
RUNNING 0 STOREN 1 2
RUNNING 1 LOAD 1
RUNNING 2 OP 0, aten::is_floating_point
RUNNING 3 JF 3
RUNNING 4 LOADC 1
RUNNING 5 JMP 3
RUNNING 8 STORE 3
RUNNING 9 MOVE 3
```
`(OP, 2, 0)` means the instruction is `OP`, and it will run the second operator in operator list. If we want to apply upgrader to this operator,  `(OP, 2, 0)` (assume this operator is `linspace_0_7`) will be replaced by a different bytecodes, which is the 7th from `getUpgraderBytecodeList()`.

Reply:
Let's add a comment here explaining the format of entries and how to add a new one so that will be clear to in the future, too

Reply:
Sounds good! will do